tools/integration_tests/local_file/read_dir_test.go

```diff
@@ -168,8 +167,8 @@ func TestReadDirWithSameNameLocalAndGCSFile(t *testing.T) {
 
 	// Attempt to list testDir.
 	_, err := os.ReadDir(testDirPath)
-	if err == nil || !strings.Contains(err.Error(), "input/output error") {
-		t.Fatalf("Expected error: %s, Got error: %v", "input/output error", err)
+	if err != nil {
```

Comment:
We also need to change here - https://github.com/GoogleCloudPlatform/gcsfuse/blob/f6ad2829c5e1504c2414042a53daff82e6531fcb/tools/integration_tests/concurrent_operations/concurrent_listing_test.go#L244

Reply:
We can remove this too - https://github.com/GoogleCloudPlatform/gcsfuse/blob/f6ad2829c5e1504c2414042a53daff82e6531fcb/tools/integration_tests/concurrent_operations/concurrent_listing_test.go#L364

As pre-creation of files don't need once this issue is fixed.

Reply:
Fixed.